### DISCUSSION TITLE
Quasar dev server cache won't clear

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#14849] [General - Components / Directives / etc] [answered]
I have a webcomponent that I use in my quasar project, and while I am developing locally I hand copy the files into the node_modules folder (whereas when installing the official version I use the npm install), and this normally has worked well, but recently Quasar is holding onto the previous build somehow, despite me doing several `quasar clean` commands and completely clearing browser caches. If I do a regular build of my PWA, the correct files are used and it works as expected. But if I am in `quasar dev` mode the old (somehow cached) version is being used. How can I really, truly clear the dev server cache?

**Comment by rstoenescu** [comment, marked as answer]
If it's a one time thing, then delete node_modules/.q-cache, otherwise there's a setting for this in quasar.config.js > build.

**Reply by ssuess** [reply]
Thanks that helped.
